lib/core/src/lib/comments/comments.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="adf-comments-container">
-    <div id="comment-header" class="adf-comments-header" role="heading">
```

Comment:
Same here

Reply:
Please check my explanation in previous comment